core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -110,3 +114,57 @@ describe('Customized profile page for current user', function() {
     general.checkForConsoleErrors([]);
   });
 });
+
+describe('Visiting user profile page',
+  function() {
+    var TEST_USERNAME = 'myUser';
+    var TEST_EMAIL = TEST_USERNAME + '@example.com';
+
+    var ANOTHER_USERNAME = 'anotherUser';
+    var ANOTHER_EMAIL = ANOTHER_USERNAME + '@example.com';
+
+    var profilePage = null;
+    var creatorDashboardPage = null;
+
+    var EXPLORATION = {
+      title: 'A new exploration',
+      category: 'Learning',
+      objective: 'The goal is to create a new exploration',
+      language: 'English'
+    };
+
+    beforeAll(function() {
+      profilePage = new ProfilePage.ProfilePage();
+      creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+
+      users.createUser(ANOTHER_EMAIL, ANOTHER_USERNAME);
+      users.login(ANOTHER_EMAIL);
+
+      workflow.createAndPublishExploration(
+        EXPLORATION.title,
+        EXPLORATION.category,
+        EXPLORATION.objective,
+        EXPLORATION.language
+      );
+
+      creatorDashboardPage.get();
+      creatorDashboardPage.expectToHaveExplorationCard(EXPLORATION.title);
+      users.logout();
+    });
+
+    it('should show the explorations created by the user',
+      function() {
```

Comment:
`    it('should show the explorations created by the user', function() {`

i.e, can we move `function() {` to the line above?